src/pages/error-page/error-page.test.js

```diff
@@ -0,0 +1,64 @@
+import userEvent from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter, useNavigate } from 'react-router-dom';
+import { ErrorPage } from './error-page';
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useAsyncError: jest.fn().mockReturnValueOnce({
+    message: 'Mock message',
+    cause: {
+      status: 400,
+      statusText: 'Mock status text',
+    },
+  }),
+  useNavigate: jest.fn(),
+}));
+
+describe('Error page', () => {
+  it('Renders status', () => {
```

Comment:
What's the difference between this test and the second one?

Reply:
The idea is that the first test accepts an error containing information about that error, and the second one is just renders the default information. Edited the tests to make it more clear.